ChatMessage.js

```diff
@@ -0,0 +1,327 @@
+const mojangson = require('mojangson')
+const vsprintf = require('sprintf-js').vsprintf
+
+module.exports = loader
+
+function loader (mcVersion) {
+  const mcData = require('minecraft-data')(mcVersion)
+  defaultLang = mcData.language
```

Comment:
Don't define `defaultLang` and `ChatMessage` in global, but put them inside the loader function. Otherwise the code will break as soon as you try instantiating multiple versions. (we had a similar issue in anvil)